onnxruntime/core/framework/op_kernel_context_internal.h

```
@@ -53,6 +53,10 @@ class OpKernelContextInternal : public OpKernelContext {
     return OpKernelContext::GetOutputMLValue(index);
   }
 
+  void SetOutputMLValue(int index, const OrtValue& ort_value) {
+    OpKernelContext::SetOutputMLValue(index, ort_value);
```

This function can silently fail since you're ignoring the return value.